cli/src/main/java/org/commonjava/maven/ext/cli/Cli.java

```diff
@@ -454,26 +455,31 @@ private Settings parseSettings( File settings ) throws ComponentLookupException,
      */
     private boolean runningInContainer()
     {
+        final Path cgroup = Paths.get( getCGroups() );
         boolean result = false;
 
-        // Don't use Paths.get as causes issues on Windows.
-        try (Stream<String> stream = Files.lines( new File( getCGroups() ).toPath() ) )
+        if ( Files.isReadable( cgroup ) )
         {
-            result = stream.anyMatch( line -> line.contains( "docker") || line.contains( "kubepods" ) );
-        }
-        catch ( IOException e )
-        {
-            logger.error( "Unable to determine if running in a container", e );
+            try ( Stream<String> stream = Files.lines( cgroup ) )
+            {
+                result = stream.anyMatch( line -> line.contains( "docker" ) || line.contains( "kubepods" ) );
+            }
+            catch ( IOException e )
+            {
+                logger.error( "Unable to determine if running in a container", e );
+            }
         }
+
         if ( !result )
         {
             result = System.getenv().containsKey( "container" );
         }
+
         return result;
     }
 
     // Split into separate method for testing only.
-    private String getCGroups()
+    private static String getCGroups()
```

Comment:
Will the byteman test still work now you have changed it to static?

Reply:
Yes, the `CliLoggingTest` passed with no changes to that BM annotation after making this change, but I could revert it if you want.

My reasoning was that the method could be static and I was a bit confused since the `getCGroups` method itself was not static, but the fake cgroup results that we return from the tests for this method are static (if I am making sense---they might not be related at all).

Reply:
Thats fine, I couldn't remember off top of my head whether the byteman would work with the change.